portfolio/portfolio/public/js/app1.js

```diff
@@ -0,0 +1,45 @@
+let http = require('http');
+let fs = require('fs');
+let url = require('url');
+let imagesArray = "png jpeg jpg".split(" ");
+http.createServer((req, res) => {
+    let parsedUrl = url.parse(req.url);
+
+    if (parsedUrl.pathname === '/') {
+
+        res.writeHead( 201, {'content-Type': 'text/html'} );
+        fs.createReadStream('../../templates/index.html').pipe(res);
+
+    } else if (parsedUrl.pathname === '/index.html') {
+
+        res.writeHead( 201, {'content-Type': 'text/html'} );
+        fs.createReadStream('../../templates/index.html').pipe(res);   
+
+    } else if (parsedUrl.pathname === '/visit') {
+
+        res.writeHead( 201, {'content-Type': 'text/html'} );
+        fs.createReadStream('../../templates/visit.html').pipe(res);   
+        
+    } else if (parsedUrl.pathname === '/register') {
+
+        res.writeHead( 201, {'content-Type': 'text/html'} );
+        fs.createReadStream('../../templates/register.html').pipe(res);   
+        
+    } else if (parsedUrl.pathname.includes ('main.css') ) {
+        res.writeHead( 201, {'content-Type': 'text/css'} );
+        fs.createReadStream('../css/main.css').pipe(res);
+
+    } else {
+        for (let i=0; i< imagesArray.length; i++) {
+            if(parsedUrl.pathname.endsWith(imagesArray[i])) {
+                res.writeHead( 201, {'content-Type': 'image/jpeg'} );
+                res.writeHead( 201, {'content-Type': 'image/jpg'} );
+                res.writeHead( 201, {'content-Type': 'image/png'} );
+                fs.createReadStream('.' + req.url).pipe(res);
+
```

Comment:
For each and every image, you are setting the content-type to all three. An image can be of only one content-type.
For example:- 
JPG/JPEG -> image/jpeg
PNG -> image/png
SVG -> image/svg

You could have handled it like:-
```js
// first grab the extension name from image url
// split the url on dot and pop the last string which is extension name for image.
var extName = req.url.split('.').pop()

// Now set the content-type like
res.setHeader('content-type', `image/${extName}`)

// then send the response like 
fs.createReadStream('.' + req.url).pipe(res);
```